package.json

```diff
@@ -47,13 +49,14 @@
     "eslint-plugin-jsdoc": "^36.1.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.3.1",
-    "jest": "^26.4.2",
+    "jest": "^28.1.0",
     "prettier": "^2.2.1",
     "prettier-plugin-packagejson": "^2.2.11",
     "rimraf": "^3.0.2",
-    "ts-jest": "^26.3.0",
+    "stdio-mock": "^1.2.0",
+    "ts-jest": "^28.0.8",
```

Comment:
Getting this peer dependency warning:

```
warning " > ts-jest@28.0.8" has incorrect peer dependency "typescript@>=4.3".
```

Reply:
I fixed this by bumping TypeScript: https://github.com/MetaMask/utils/pull/21

Reply:
This is merged now.